### ISSUE TITLE
Folder structure

### ISSUE THREAD

**Issue by denizs** [EndingCredits/gym-utils#6]
Maybe we should start introducing subfolders. If we continue to add wrappers, our `gym_utils` folder will blow up.
I'd propose something along the lines of:
* gym_utils
  * wrappers
  * base_agents

**Comment by EndingCredits** [comment]
Yeah, the module structure is something we'll need to look at. I think we can leave it until after we have all the basic stuff done so we know what stuff we need to sort out.
